rclcpp/test/rclcpp/test_qos_event.cpp

```diff
@@ -158,6 +173,11 @@ std::string * g_sub_log_msg;
 std::promise<void> * g_log_msgs_promise;
 TEST_F(TestQosEvent, test_default_incompatible_qos_callbacks)
 {
+  std::string rmw_implementation_str = std::string(rmw_get_implementation_identifier());
```

Comment:
This event is fully supported by `rmw_zenoh` so we shouldn't have to skip it. if you're getting a test failure here because of mismatched expectations, we should update the test similar to 4) in https://github.com/ros2/rcl/pull/1164